src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -112,7 +97,7 @@ class Wav2Vec2EncoderConfig:
     sample_fbank_every_k: int
 
     # Position Encoder
-    pos_encoder_type: Literal["conv", "relative", "relative_shaw", "rotary"]
+    pos_encoder_type: str
```

Comment:
Nit: Can you update the docstr to something like: "The type of position encoder ('conv', 'relative', 'rotary')."